app/controllers/uploading_controller.rb

```diff
@@ -16,6 +16,7 @@ def set_s3_url
       success_action_status: '201',
       acl: 'public-read',
       content_type_starts_with: 'image/',
-      cache_control: 'public, max-age=31536000')
+      cache_control: 'public, max-age=31536000',
+    )
```

Comment:
Think this is from the comma PR but it's not like it matters :D

Reply:
The comma must have snuck in when I was rebasing xD